### PR TITLE
Auditissue

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
pls review

### CODE REVIEW

File: src/components/Header/ConnectionStatus.js [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
+    this.state = {
+      connectionStatus:''
+    };
+  }
+
+
+  componentWillMount(){
+    if (window && window.navigator.onLine) {
+      this.state.connectionStatus = 'Connected.';
+      // console.log('online','=====================');
```
Comment (ShankyRana) [inline src/components/Header/ConnectionStatus.js]: Remove log

File: src/components/Dashboard/Dashboard.scss [hunk 2]
```diff
@@ -17,7 +17,7 @@
   p{
     &.text-uppercase {
       &.title {
-        min-height: 30px;
+        //min-height: 30px;
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.scss]: Remove

File: src/ducks/configuration.js [hunk 3]
```diff
@@ -65,11 +65,22 @@ const initialConfig = {
     recentConv: 5,
     hideAnonymousConversion: true,
     onlyDisplayNotification: false,
-    liveVisitorCount: 0
+    liveVisitorCount: 0,
```
Comment (ShankyRana) [inline src/ducks/configuration.js]: Remove trailing comma

File: src/components/WebsiteDemoPage/WebsiteDemoPage.js [hunk 4]
```diff
@@ -16,6 +18,12 @@ const initialState = {
   fm1 : true,
   fm2 : false,
   fm3 : false,
+  name: '',
+  isEmailValid: false,
+  errorEmail: '',
+  errorFirstName: '',
+  errorLastName: '',
+  errorPhoneNumber: '',
```
Comment (ShankyRana) [inline src/components/WebsiteDemoPage/WebsiteDemoPage.js]: Trailing comma

File: src/components/WebsiteDemoPage/WebsiteDemoPage.js [hunk 5]
```diff
@@ -48,6 +85,14 @@ class WebsiteDemoPage extends  Component  {
     }));
   }
 
+  // validateForm(){
```
Comment (ShankyRana) [inline src/components/WebsiteDemoPage/WebsiteDemoPage.js]: Remove commented code

File: src/components/Header/ConnectionStatus.js [hunk 6]
```diff
@@ -15,10 +15,10 @@ export default class ConnectionStatus extends Component{
   componentWillMount(){
     if (window && window.navigator.onLine) {
       this.state.connectionStatus = 'Connected.';
-      // console.log('online','=====================');
+      console.log('online','=====================');
```
Comment (ShankyRana) [inline src/components/Header/ConnectionStatus.js]: Remove

File: src/components/Header/ConnectionStatus.css [hunk 7]
```diff
@@ -55,7 +57,7 @@
     display: none;
 }
 .center-block {
-  display: block;
+  /* display: block; */
```
Comment (ShankyRana) [inline src/components/Header/ConnectionStatus.css]: Remove

File: src/components/ResetPassword/index.js [hunk 8]
```diff
@@ -66,24 +62,28 @@ export default class forget extends Component{
...
   }
-
   handleSubmit(evt){
-    evt.preventDefault();
-    var token = document.location.href.split('token=')[1];
-    const data = {
-      'newPassword' :  this.state.newPassword,
-      'verifyPassword': this.state.verifyPassword,
-      'token': token
+    if (!this.canBeSubmitted()) {
```
Comment (ShankyRana) [inline src/components/ResetPassword/index.js]: Test this. If it's working or not

File: src/components/Dashboard/Dashboard.scss (hunk 2 above)
Comment (Devilla) [reply src/components/Dashboard/Dashboard.scss]: done

File: src/components/ResetPassword/index.js (hunk 8 above)
Comment (Devilla) [reply src/components/ResetPassword/index.js]: yup working

**Comment by Devilla** [comment]
Merge it!

**Pull request merged**